### PR TITLE
virt-api: run container as non-root user

### PR DESCRIPTION
**Pull request by cynepco3hahue**
Create new virt-api user and run all commands under this user.
Part of the fix for #113

### CODE REVIEW

**Comment by cynepco3hahue** [comment]
@rmohr Can you please check the patch? I can see under Travis that test fail does not connect to the patch.

File: cmd/virt-api/Dockerfile [hunk 1]
```diff
@@ -18,11 +18,17 @@
 
 FROM centos:7
 
+# Create non-root user
+RUN useradd --create-home -s /bin/bash virt-api
+WORKDIR /home/virt-api
+USER virt-api
```
Comment (rmohr) [inline cmd/virt-api/Dockerfile]: Does it make sense to switch the USER after the RUN command below?

**Comment by rmohr** [comment]
@cynepco3hahue yes was a transient error. Thanks for doing this :+1:

File: cmd/virt-api/Dockerfile (hunk 1 above)
Comment (cynepco3hahue) [inline cmd/virt-api/Dockerfile]: RUN useradd - create new user
USER virt-api - change execution user to virt-api, so all commands after this one will be executed with user virt-api
so if you want to include
securityContext:
        runAsNonRoot: true
to the manifest yaml you musr run this command in the dockerfile

I believe I can include this lines in the end of the Dockerfile and do not change all swagger commands(I believe I will do it this way for the haproxy container), give me know if you prefer it this way

**Comment by rmohr** [comment]
Want to test it before merging, but looks good.

**Pull request merged**